endpoint/phonebook.go

```diff
@@ -76,8 +77,9 @@ func MakeGetDetail(ctx context.Context, usecase usecase.Provider) endpoint.Endpo
 			Village:        resp.Village,
 			Latitude:       resp.Latitude,
 			Longitude:      resp.Longitude,
-			CoverImagePath: resp.CoverImagePath,
+			CoverImagePath: fmt.Sprintf("%s/%s", cfg.AppStoragePublicURL, resp.CoverImagePath),
 			Status:         resp.Status,
+			StatusLabel:    GetStatusLabel[resp.Status]["ina"],
```

Comment:
@setiadijoe Some leftovers :grin: 
```suggestion
			StatusLabel:   GetStatusLabel[v.Status]["id"],
```

Reply:
i had created on new commit